agent/internal/agent.go

```diff
@@ -102,9 +102,9 @@ func (a *Agent) run(ctx context.Context) error {
 	case msg, ok := <-socket.Inbox:
 		switch {
 		case !ok:
-			return fmt.Errorf("socket closed while reading setup messages")
+			return errors.New("socket closed while reading setup messages")
```

Comment:
The plan is to get rid of `github.com/pkg/errors` in the near future, so this kind of change seems like it shouldn't happen. Alternatively, the stdlib `errors` package has a function with the same name and behavior that could be used.

Reply:
sounds good, I'll disable this linter